serviceAccount/README.md

```diff
@@ -0,0 +1,90 @@
+
+2. Import the PowerShell Module on all servers
+
+    In a PowerShell window, run the following:
+
+    ```ipmo ServiceAccount.psm1```
+
+3. For Windows Server 2016 and later, add a rule granting the new service account necessary permissions.
+
+	In a PowerShell window on the primary AD FS server, run the following:
+
+    ```Add-AdfsServiceAccountRule -ServiceAccount <ServiceAccount> -SecondaryServers <ListOfSecondaryServers>```
+	Note that ```<ServiceAccount>``` should be the serivce account you want to grant permissions to and can be provided either in the format Domain\User or merely User.
+	```<ListOfSecondaryServers>``` should be replaced with a list of secondary servers if the environment is a WID farm so that the configuration database can be synced across all machines.
+	
+4. Change the service account on each machine in the farm.
+
+	Beginning with the secondary servers, run the following:
+	
+	```Update-AdfsServiceAccount```
+	Once the function has been executed on all secondary servers, proceed to run it on the primary server.
+	
+5. If Device Registration Services (DRS) is set up in your AD FS environment, you must also use the ```Set-AdfsDeviceRegistration``` cmdlet (an internal command exposed by the service) to add the proper permissions to the new service account.
+
+	
+6. For Windows Server 2016 and later, remove the rule granting permissions to the old service account.
+
+	In a PowerShell window on the primary AD FS server, run the following:
+
+    ```Add-AdfsServiceAccountRule -ServiceAccount <ServiceAccount> -SecondaryServers <ListOfSecondaryServers>```
+	Note that ```<ServiceAccount>``` should be the serivce account you want to revoke permissions for and can be provided either in the format Domain\User or merely User.
+	```<ListOfSecondaryServers>``` should be replaced with a list of secondary servers if the environment is a WID farm so that the configuration database can be synced across all machines.
+	
+	
+
+
+## Add-AdfsServiceAccountRule Parameters
+
+__`-ServiceAccount`__ - Name of the service account for which to add a new rule. Can be provided in either the format Domain\User or User
+
+__`-SecondaryServers`__ - Comma separated list of AD FS secondary servers. List is used to force a WID sync across all machines in the farm.
+
+## Remove-AdfsServiceAccountRule Parameters
+
+__`-ServiceAccount`__ - Name of the service account for which to remove permissions rule. Can be provided in either the format Domain\User or User
+
+__`-SecondaryServers`__ - Comma separated list of AD FS secondary servers. List is used to force a WID sync across all machines in the farm.
+
+## Restore-AdfsSettingsFromBackup Parameters
+
+__`-BackupPath`__ - Path to backup file generated by either Add-AdfsServiceAccountRule or Remove-AdfsServiceAccountRule. Should reseble C:\Users\Administrator\Documents\serviceSettingsData-2018-04-11-12-04-03.xml
+
+## Additional Details
```

Comment:
I would rename this section to something like "Warning Before Use", and move it up to be right after the "Overview" section